pom-gwt.xml

```diff
@@ -122,41 +122,16 @@
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-user</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
 
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-dev</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
   </dependencies>
 
-  <repositories>
```

Comment:
It's published at https://oss.sonatype.org/content/repositories/snapshots/com/google/guava/guava/20.0-SNAPSHOT/